node/cn/handler_test.go

```diff
@@ -61,8 +65,14 @@ func init() {
 		nodeids[i] = discover.PubkeyID(&keys[i].PublicKey)
 	}
 
-	tx = types.NewTransaction(111, addrs[0], big.NewInt(111), 111, big.NewInt(111), nil)
-	txs = types.Transactions{tx}
+	signer := types.MakeSigner(params.BFTTestChainConfig, big.NewInt(2019))
```

Comment:
2nd argument is not used in the function. I was used as a block number.

```suggestion
	signer := types.MakeSigner(params.BFTTestChainConfig, nil)
```

Reply:
Unless we change the function `types.MakeSigner`, I think there's no need to give `nil` to the second parameter. If so, we'd better create a PR to change the interface and related usages. How do you think?

Reply:
Yes, we need to clean the function. Anyway, 2nd parameter is not chain id, it is block number. So I thought the 2019 is weird.

Reply:
Anyway. In next PR, let's clean the function. :)